darknet.py

```diff
@@ -441,5 +452,72 @@ def performDetect(imagePath="data/dog.jpg", thresh= 0.25, configPath = "./cfg/yo
             print("Unable to show image: "+str(e))
     return detections
 
+def performBatchDetect(thresh= 0.25, configPath = "./cfg/yolov3.cfg", weightPath = "yolov3.weights", metaPath= "./cfg/coco.data", hier_thresh=.5, nms=.45, batch_size=3):
+    import cv2
+    import numpy as np
+    # NB! Image sizes should be the same
+    # You can change the images, yet, be sure that they have the same width and height
+    img_samples = ['data/person.jpg', 'data/person.jpg', 'data/person.jpg']
```

Comment:
Why do you use 3 the same images, instead of `dog.jpg, person.jpg, eagle.jpg` ?
Since all these images will be resized to the same network size - you can use many **different** images.

Reply:
It's been a while so I don't recall the exact reason but it could be due to that network corrects predictions in the end according to the original input size. In the sequential inference case, it's obvious that there's only one image at a time so it doesn't matter if the image sizes vary. For instance, consider the following function call. **w** and **h** are the same for all the input images.
`correct_yolo_boxes(dets, count, w, h, netw, neth, relative, letter);`

Reply:
And, yes run-times include both pre and post-processing along with prediction.